app/views/public/event_detail.py

```diff
@@ -141,8 +141,9 @@ def display_event_schedule(identifier):
         abort(404)
     tracks = DataGetter.get_tracks(event.id)
     accepted_sessions_count = get_count(DataGetter.get_sessions(event.id))
+    confirmed_sessions_count = get_count(DataGetter.get_sessions(event.id, state='confirmed'))
```

Comment:
NO! No ... Please no. Why can't you do the way I suggested in the previous comment ? (regarding the `get_sessions` method)